animareflection/ui
# Bun Migration

## Description

Migrated from yarn to bun.

## Test Steps

1) Verify that all scripts in `package.json` still run and work appropriately.
2) Verify that there are no references to `yarn` in the codebase.
3) Verify that CI workflows work.
